app/classes/report/symbiota.rb

```diff
@@ -2,7 +2,14 @@
 
 module Report
   # Symbiota-style tsv report.
+  # https://symbiota.org/
+  # https://biokic.github.io/symbiota-docs/
+  # https://github.com/Symbiota/Symbiota
   class Symbiota < TSV
+    # These plus Column labels comprising
+    # Symbiota Standard Field names which are useful for our purposes
+    # https://biokic.github.io/symbiota-docs/editor/edit/fields/#standard-fields
+    # plus MO-specific fields that are useful for uploads to Symbiota portals
     def labels
```

Comment:
I'd like to standardized the labels to be Symbiota field names (if possible).
Proposed changes
- `host` (obsolete) => `associated_taxa` ([the official Symbiota field name](https://biokic.github.io/symbiota-docs/editor/edit/fields/#associatedTaxa)).
- `fieldNotes` (a description of the contents) => [`occurrenceRemarks`](https://biokic.github.io/symbiota-docs/editor/edit/fields/#occurrenceRemarks) 
- `mushroomObserverId` (not a Symbota field) => [`recordNumber`](https://biokic.github.io/symbiota-docs/editor/edit/fields/#recordNumber)
-  Also add [`fieldnumber`](https://biokic.github.io/symbiota-docs/editor/edit/fields/#fieldnumber) with a value of collector's number / field slip number